script/script.go

```diff
@@ -264,6 +267,16 @@ func serveTableWalk(res *http.Response) func(lua.LValue, lua.LValue) {
 			h := make(http.Header)
 			t.ForEach(serveHeaderWalk(h))
 			res.Header = h
+
+		case "body":
+			// would love to be able to pass a table to be then rendered as
+			// json body... but layeh.com/gopher-json does not export toJSON
```

Comment:
I hope you created an issue for that upstream :)

Reply:
not yet... will do a PR later

Reply:
upstream PR https://github.com/layeh/gopher-json/pull/2